.github/workflows/codeql.yml

```diff
@@ -82,16 +98,28 @@ jobs:
       with:
         distribution: corretto
         java-version: '11'
-      if: ${{ matrix.language == 'java' }}
+      if: ${{ matrix.target.language == 'java' }}
 
     - name: build Java
       uses: gradle/gradle-build-action@v2
       with:
         arguments: build --no-build-cache
-        build-root-directory: java
-      if: ${{ matrix.language == 'java' }}
+        build-root-directory: ${{ matrix.target.directory }}
+      if: ${{ matrix.target.language == 'java' }}
+
+    - name: setup dotnet
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 6.x
+      if: ${{ matrix.target.language == 'csharp' }}
+
+    - name: build dotnet
+      # Build all dotnet applications from this directory
+      run: find . -name '*.sln' -exec dotnet build '{}' \;
+      working-directory: ${{ matrix.target.directory }}
```

Comment:
Is this still needed w/ line 82?

Reply:
this could be removed. I preferred to leave it here just to be consistent with the other languages. (the find command will be executed from the dotnet directory).